examples/gltf/app.js

```diff
@@ -331,21 +346,21 @@ export class DemoApp {
     switch (value) {
       case 'exclusive':
         Object.assign(this.loadOptions, {
-          pbrIbl: true,
+          imageBasedLighting: this.environment,
```

Comment:
Is there any reason not to just call this prop `environment`?

* `imageBasedLighting` sounds like a boolean.
* `imageBadedLightingEnvironment`?

Reply:
imageBasedLightingEnvironment

Reply:
it is a **specific** type of environment